extensions/logging-gelf/runtime/src/main/java/io/quarkus/logging/gelf/GelfConfig.java

```diff
@@ -93,4 +93,9 @@
     @ConfigDocSection
     public Map<String, AdditionalFieldConfig> additionalField;
 
+    /**
+     * Whether to include all fields from the {@link org.slf4j.MDC}
```

Comment:
```suggestion
     * Whether to include all fields from the MDC.
```

We don't want a reference to the SLF4J API as we don't use it (we use JBoss Logging).